hyperion-geiger-counter/src/main/java/com/willowtreeapps/hyperion/geigercounter/GeigerCounterModule.java

```diff
@@ -30,6 +38,33 @@ private void updateView(boolean detectorIsEnabled) {
         }
     }
 
+    private void showMenu(View anchor) {
+        final Context context = anchor.getContext();
+        if (popupMenu != null) {
+            popupMenu.dismiss();
```

Comment:
Probably you want this popup to dismiss if the menu becomes closed. I would check out the PhoenixModule; it monitors menu state changes for this purpose.